README.md

```diff
@@ -128,17 +129,31 @@ In addition, the provider can be configured using attributes in the
 configuration. Attributes specified in `main.tf` override the configuration loaded by
 `conjur-api-go`.
 
-For example, if the environment is initialized as above, this configuration would
-authenticate as `terraform-user` instead of `admin`:
+For example, with `conjur_api_key` and `conjur_ssl_cert`defined as
+[input variables](https://www.terraform.io/docs/configuration/variables.html), this
+type of configuration could be used:
 
 ```
 # main.tf
+variable "conjur_api_key" {}
+variable "conjur_ssl_cert" {}
+
 provider "conjur" {
+  appliance_url = "http://conjur-server"
+  ssl_cert = var.conjur_ssl_cert
+  # If you have the certificate as a file, use this line instead
+  # ssl_cert_path = var.conjur_ssl_cert_path
+
+  account = "myorg"
+
   login = "terraform-user"
-  api_key = "x0dwqc3jrqkye3xhn7k62rw31c6216ewfe1wv71291jrqm4j15b3dg9"
+  api_key = var.conjur_api_key
 }
 ```
 
+_Note as well that `login`/`api_key` settings are only used when both are present in
+the attributes otherwise the provider will fall back to using environment variables
+for those two values_
```

Comment:
is this implying that if only one is provided, it will override it with the env vars?

that seems weird

it makes sense to me that one has to take precedence if both are set, but it should apply to all variables equally - and I'm not sure why I shouldn't be allowed to set my API key as an env var but set the login here

Reply:
(it should also follow whatever standard we've set in our other projects, for when env var and literal config are both set)

Reply:
> it makes sense to me that one has to take precedence if both are set, but it should apply to all variables equally - and I'm not sure why I shouldn't be allowed to set my API key as an env var but set the login here

Yeah it's weird but it's due to the quirky design of the api that requires both variables present to make a "LoginPair". Decoupling would have meant that I had to import a ton of code from the API. Do note that this isn't a result of my change here but has been just undocumented behavior of the original provider code.